syncer/db.go

```diff
@@ -159,6 +202,16 @@ func (conn *Conn) executeSQLWithIgnore(tctx *tcontext.Context, ignoreError func(
 		FirstRetryDuration: retryTimeout,
 		BackoffStrategy:    retry.Stable,
 		IsRetryableFn: func(retryTime int, err error) bool {
+			if retry.IsConnectionError(err) {
+				err := conn.resetConn(tctx)
+				if err != nil {
+					tctx.L().Warn("reset connection failed", zap.Int("retry", retryTime),
+						zap.String("queries", utils.TruncateInterface(queries, -1)),
+						zap.String("arguments", utils.TruncateInterface(args, -1)))
```

Comment:
log error also

Reply:
Done in [here](https://github.com/pingcap/dm/pull/266/files#diff-c3cc6fccd12f8e1679a575d1aaf8fd14R213)